static/styles/portico/portico.scss

```diff
@@ -1766,3 +1766,21 @@ input.new-organization-button {
 .unsupported_browser_page {
     padding-bottom: 80px;
 }
+
+#navbar-custom-message {
+    font-size: 1rem;
+    background-color: hsl(174, 82%, 28%);
```

Comment:
We can work on improving the alert background color in subsequent PRs if necessary.